### PR TITLE
#177 working checking if operation is available

### PR DESCRIPTION
**Pull request by Shaihuludus**

### CODE REVIEW

File: bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/sidekick/AemSidekick.java [hunk 1]
```diff
@@ -161,6 +165,14 @@ public AemSidekick clickCustomOperation(String optionName) {
     return this;
   }
 
+  public boolean isPageOperationEnabled(PageOperation optionName){
+    return isOperationEnabled(getOperation(optionName));
+  }
+
+  public boolean isCustomOperationEnabled(String optionName){
```
Comment (mkrzyzanowski) [inline bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/sidekick/AemSidekick.java]: Javadocs on public methods please :)

File: bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/sidekick/AemSidekick.java [hunk 2]
```diff
@@ -363,6 +375,14 @@ public boolean isCollapsed() {
     return map;
   }
 
+  private boolean isOperationEnabled(WebElement operation){
+    if(operation != null){
```
Comment (mkrzyzanowski) [inline bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/sidekick/AemSidekick.java]: Null check is redundant, `findElement()` will never return a null - the whole expression can be simplified.

Comment (Shaihuludus) [inline bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/sidekick/AemSidekick.java]: done

File: bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/sidekick/AemSidekick.java (hunk 1 above)
Comment (Shaihuludus) [inline bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/sidekick/AemSidekick.java]: done

**Pull request merged**